modules/partition-distribution/src/main/java/org/apache/ignite/internal/partitiondistribution/PartitionDistributionUtils.java

```diff
@@ -17,19 +17,21 @@
 
 package org.apache.ignite.internal.partitiondistribution;
 
+import static java.util.Collections.emptyList;
 import static java.util.stream.Collectors.toList;
 import static java.util.stream.Collectors.toSet;
 
-import java.util.ArrayList;
 import java.util.Collection;
-import java.util.HashSet;
 import java.util.List;
 import java.util.Set;
 
 /**
  * Stateless distribution utils that produces helper methods for an assignments distribution calculation.
  */
 public class PartitionDistributionUtils {
+
+    private static final DistributionAlgorithm DISTRIBUTION_ALGORITHM = new RendezvousDistributionFunction();
```

Comment:
Are we going to add an ignite system property at least here?

Reply:
For now, it's not needed

Reply:
If we have a plan to support different affinity algorithms, we need to leave a TODO here.